docs/pages/4 - Tasks.md

```diff
@@ -244,6 +242,14 @@ yourself repeating in `Target`s and `Command`s.
 def downstreamTarget = T { ... foo() ... } 
 def downstreamCommand = T.command { ... foo() ... } 
```

Comment:
This is not a valid command, too.

Reply:
The following code works 
```scala
import mill._
def foo() = T.task { println(42)}
def t2() = T.command { foo() }
```

Reply:
Sorry, what I meant is the `downstreamCommand` is missing the `()`

Reply:
Use the old example with an argument.